src/eva/diagnostics/scatter.py

```diff
@@ -1,23 +1,55 @@
 from eva.eva_path import return_eva_path
+from eva.utilities.config import get
 from eva.utilities.utils import get_schema, update_object
 import eva.plot_tools.plots
 import os
+import numpy as np
 
 
 class Scatter():
 
     def __init__(self, config, logger, dataobj):
-        # prepare data based on config
+
+        # Get the data to plot from the data_collection
+        # ---------------------------------------------
         varnames = config['comparison']
-        xdata = dataobj.get_variable_data(varnames[0])
-        ydata = dataobj.get_variable_data(varnames[1])
-        # create declarative plotting Scatter object
+        var0 = varnames[0]
```

Comment:
FYI, I have in my branch a change that instead of `comparison` and then a list of vars it is now two keys `x:` and `y:` which I think is more explicit for scatter plots.

Reply:
OK, good to know. We can update the YAMLs. That makes more sense.